_resources/_styles/type.styl

```diff
@@ -140,19 +140,39 @@ h3, h4
 		left -.4em
 
 
-//POST TYPE
-
-.post
-	h3, h4
-		font-family typeface-h3
-		font-size 2vw
-		margin-top 2rem
-	strong, em
-		font-family typeface-h3
-		margin 2em 0
-		display block
-	sup
-		font-size 50%
-		top -1em
-	ol
-		opacity .5
+// POSTS
+	
+.postContent h1 {
```

Comment:
Consistency in code is a lot more important than perfection. I'm using thing kind of  CamelCase on all classes on this project so let's make it `PostContent`